lua/entities/gmod_wire_cd_disk.lua

```diff
@@ -33,7 +33,11 @@ function ENT:BuildDupeInfo()
 		info["DiskMemory"][k] = dataptr
 		info["DiskData"..dataptr] = {}
 		for k2,v2 in pairs(self.DiskMemory[k]) do
-			info["DiskData"..dataptr][k2] = v2
+			if isnumber(v2) then
+				info["DiskData"..dataptr][k2] = v2
+			else
+				info["DiskData"..dataptr][k2] = 0
+			end
```

Comment:
Can you do `info["DiskData"..dataptr][k2] = isnumber(v2) or 0`?

Reply:
It says [isnumber](https://wiki.facepunch.com/gmod/Global.isnumber) returns bool, would something an expression like `info["DiskData"..dataptr][k2] = isnumber(v2) or 0` change that?